server/src/main/java/com/flipkart/varadhi/db/ZNode.java

```diff
@@ -14,6 +14,7 @@
  * <ul>
  *     <li>Entities: /varadhi/entities/[kind]/[name]</li>
  *     <li>Hierarchical Resources: /varadhi/entities/[kind]/[parent]:[name]</li>
+ *     <li>Child Resources: /varadhi/entities/[kind]/[parent]/[kind]/[name]</li>
```

Comment:
[kind]/[parent]/[kind]/[name] looks wrong.

the namedFilters path should look like 
`/varadhi/entities/Org/my_org/Filters` or `/varadhi/entities/Org/my_org/NamedFilters`